test/e2e/onINP-test.js

```diff
@@ -651,6 +651,30 @@ describe('onINP()', async function () {
       assert.equal(inp1.attribution.interactionTarget, 'html>body>main>h1');
     });
 
+    it('reports the interaction target when target is removed from the DOM', async function () {
+      if (!browserSupportsINP) this.skip();
+
+      await navigateTo('/test/inp?attribution=1&mouseup=100&click=50', {
+        readyState: 'interactive',
+      });
+
+      const button = await $('#reset');
+      await simulateUserLikeClick(button);
+
+      await nextFrame();
```

Comment:
Can we be sure that event timing will have run by the next frame?

Reply:
I don't think we can be sure, but it worked 100% of the time in my testing, and there are 2 retries if it fails.

My preference would be to use something like this rather than an arbitrary timeout. The most robust solution would be to add a PerformanceObserver and wait for the entries to dispatch, but I think what we have now is sufficient (and we can update it if we notice flakiness).

Reply:
Fair enough. It's only a test so our problem if it doesn't work.